component/src/main/java/com/iluwatar/component/App.java

```diff
@@ -0,0 +1,27 @@
+package com.iluwatar.component;
+
+import com.iluwatar.component.enums.DirectionEnum;
+import com.iluwatar.component.impl.BjornDemoInputComponent;
+import com.iluwatar.component.impl.BjornGraphicsComponent;
+import com.iluwatar.component.impl.BjornPhysicsComponent;
+import com.iluwatar.component.impl.BjornPlayerInputComponent;
+import com.iluwatar.component.objects.GameObject;
+
```

Comment:
Add description here how the code example fulfills the pattern. See for example here https://github.com/iluwatar/java-design-patterns/blob/master/adapter/src/main/java/com/iluwatar/adapter/App.java